applications/asset_tracker_v2/src/modules/debug_module.c

```diff
@@ -0,0 +1,288 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr.h>
+#include <logging/log.h>
+#if defined(CONFIG_MEMFAULT)
```

Comment:
defined(CONFIG_MEMFAULT) -> !defined(CONFIG_DEBUG_MODULE_USE_CUSTOM_TRANSPORT) ?

I am assuimg !DEBUG_MODULE_USE_CUSTOM_TRANSPORT implies CONFIG_MEMFAULT

Reply:
Actually `CONFIG_DEBUG_MODULE_USE_CUSTOM_TRANSPORT` depends on `CONFIG_MEMFAULT`. `CONFIG_DEBUG_MODULE_USE_CUSTOM_TRANSPORT` selects what transport Memfault data will be sent on. Ill update the Kconfig name to make this more clear

Reply:
Ill update it to `DEBUG_MODULE_MEMFAULT_USE_EXTERNAL_TRANSPORT` if no one disagrees.

Reply:
Seems a bit wordy. But I cant think of any other better alternative. So go for it.